lib/ocpp/common/websocket/websocket_tls.cpp

```diff
@@ -1,11 +1,12 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
-#include <boost/optional/optional.hpp>
-
 #include <everest/logging.hpp>
 
 #include <ocpp/common/pki_handler.hpp>
 #include <ocpp/common/websocket/websocket_tls.hpp>
+#include <ocpp/helpers/uri.hpp>
+
+#include <boost/optional/optional.hpp>
```

Comment:
where do we need this header? we moved to std::optional a while ago

Reply:
Not needed. I guess it was included somehow by the IDE.